components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -82,10 +85,14 @@ std::string GetPlatformIdentifier() {
 std::string BuildReferralEndpoint(const std::string& path) {
   std::unique_ptr<base::Environment> env(base::Environment::Create());
   std::string referral_server;
+  std::string proto = "https";
   env->GetVar("BRAVE_REFERRALS_SERVER", &referral_server);
   if (referral_server.empty())
     referral_server = kBraveReferralsServer;
-  return base::StringPrintf("https://%s%s", referral_server.c_str(),
+  if (env->HasVar("BRAVE_REFERRALS_LOCAL"))
+    proto = "http";
```

Comment:
Can this just be implied when `BRAVE_REFERRALS_SERVER` is set? If not, would perhaps rename to `BRAVE_REFERRALS_USE_HTTP` for clarity.

Reply:
It can't, but i would just prefer if we could stick the protocol as in as part of that variable